### PR TITLE
Add documentation for queries pagination

### PR DESCRIPTION
**Pull request by kamilsa**
Signed-off-by: kamilsa <kamilsa16@gmail.com>

### Description of the Change

Update documentation with new query pagination logic

### Benefits

Up to date documentation

### Possible Drawbacks 

None

### CODE REVIEW

File: docs/source/api/queries.rst [hunk 1]
```diff
@@ -203,15 +203,23 @@ Get Account Transactions
 Purpose
 -------
 
-In a case when a list of transactions per account is needed, `GetAccountTransactions` query can be formed.
+In a case when a list of transactions per account is needed, `GetAccountTransactions` query can be formed. Please note, that this query uses pagination for query responses.
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
In a case when a list of transactions per account is needed, `GetAccountTransactions` query can be formed. 
.. note:: Please note, that this query uses pagination for query responses.
```

File: docs/source/api/queries.rst [hunk 2]
```diff
@@ -222,14 +230,20 @@ Request Structure
     :widths: 15, 30, 20, 15
 
     "Account ID", "account id to request transactions from", "<account_name>@<domain_id>", "makoto@soramitsu"
+    "Page size", "size of the page to be returned by the query, if response contains less transactions than page size, then next tx hash will be empty in response", "page_size > 0", "5"
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
    "Page size", "size of the page to be returned by the query, if the response contains fewer transactions than a page size, then next tx hash will be empty in response", "page_size > 0", "5"
```

File: docs/source/api/queries.rst [hunk 3]
```diff
@@ -222,14 +230,20 @@ Request Structure
...
+    "First tx hash", "hash of the first transaction in the page. If that field is not set, then first transactions are returned", "hash in hex format", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
    "First tx hash", "hash of the first transaction in the page. If that field is not set — then the first transactions are returned", "hash in hex format", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```

File: docs/source/api/queries.rst [hunk 4]
```diff
@@ -240,23 +254,33 @@ Response Structure
     :widths: 15, 30, 20, 15
 
     "Transactions", "an array of transactions for given account", "Committed transactions", "{tx1, tx2…}"
+    "All transactions size", "total number of transactions created by given account", "", "100"
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
    "All transactions size", "total number of transactions created by the given account", "", "100"
```

File: docs/source/api/queries.rst [hunk 5]
```diff
@@ -240,23 +254,33 @@ Response Structure
...
+    "Next transaction hash", "hash pointing to the next transaction after the last transaction in the page. Empty if page contains the last transaction for requested account", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
    "Next transaction hash", "hash pointing to the next transaction after the last transaction in the page. Empty if a page contains the last transaction for the requested account", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```

File: docs/source/api/queries.rst (hunk 3 above)
Comment (nickaleks) [inline docs/source/api/queries.rst]: Maybe write that if no hash is specified, return transactions **starting from the first**

File: docs/source/api/queries.rst [hunk 6]
```diff
@@ -268,14 +292,20 @@ Request Structure
 
     "Account ID", "account id to request transactions from", "<account_name>@<domain_id>", "makoto@soramitsu"
     "Asset ID", "asset id in order to filter transactions containing this asset", "<asset_name>#<domain_id>", "jpy#japan"
+    "Page size", "size of the page to be returned by the query, if response contains less transactions than page size, then next tx hash will be empty in reponse", "page_size > 0", "5"
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
    "Page size", "size of the page to be returned by the query, if the response contains fewer transactions than a page size, then next tx hash will be empty in response", "page_size > 0", "5"
```

File: docs/source/api/queries.rst [hunk 7]
```diff
@@ -268,14 +292,20 @@ Request Structure
...
+    "First tx hash", "hash of the first transaction in the page. If that field is not set, then first transactions are returned", "hash in hex format", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
    "First tx hash", "hash of the first transaction in the page. If that field is not set — then the first transactions are returned", "hash in hex format", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```

File: docs/source/api/queries.rst [hunk 8]
```diff
@@ -286,6 +316,8 @@ Response Structure
     :widths: 15, 30, 20, 15
 
     "Transactions", "an array of transactions for given account and asset", "Committed transactions", "{tx1, tx2…}"
+    "All transactions size", "total number of transactions for given account and asset", "", "100"
+    "Next transaction hash", "hash pointing to the next transaction after the last transaction in the page. Empty if page contains the last transaction for given account and asset", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
    "Next transaction hash", "hash pointing to the next transaction after the last transaction in the page. Empty if a page contains the last transaction for given account and asset", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```

File: docs/source/api/queries.rst [hunk 9]
```diff
@@ -240,23 +254,33 @@ Response Structure
...
+    "Next transaction hash", "hash pointing to the next transaction after the last transaction in the page. Empty if page contains the last transaction for requested account", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
 
 Get Account Asset Transactions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Purpose
 -------
 
-`GetAccountAssetTransactions` query returns all transactions associated with given account and asset.
+`GetAccountAssetTransactions` query returns all transactions associated with given account and asset. Please note, that this query uses pagination for query responses.
```
Comment (neewy) [inline docs/source/api/queries.rst]:
```suggestion
`GetAccountAssetTransactions` query returns all transactions associated with given account and asset. 

.. note:: Please note, that this query uses pagination for query responses.
```

**Pull request merged**